crates/hirola-core/src/mixins.rs

```diff
@@ -163,3 +213,256 @@ pub mod model {
         Model(s.clone(), PhantomData)
     }
 }
+
+
+///Mixin for reactive classes
+/// 
+/// ## Example
+/// ```
+/// pub fn yellow_or_black_box(_app: &HirolaApp) -> Dom {
+///     let state = Signal::new(vec!["black-box".to_string()]);
```

Comment:
Something similar to 
```rust
use semester::{classes, Classes as _};

let classes = classes!(
    "always",
    "yup": 10 == 10,
    "nope": 10 == 15,
);

assert_eq!(classes.render(), "always yup");
```
See https://lib.rs/crates/semester

Then one could just pass `Signal` in the bool parts right?

Reply:
yep2, did tried it, care to check it out?